tspng/extraction.py

```diff
@@ -18,9 +18,15 @@ def extract_from_file(path: str, mime_type: str=MIME_TYPE) -> Dict:
         Returns:
                 dict (dict): Dictionary containing file metadata
     '''
+    #checks if path exists
+    if os.path.exists(path)==False:
```

Comment:
This can be refactor to `if not os.path.exists(path):`, which reads a little better.

Reply:
Changes made as requested